### DISCUSSION TITLE
Type error with sendParent in xstate 5.8.0

### DISCUSSION THREAD

**Discussion by scott-mfv** [statelyai/xstate#4770] [Q&A] [answered]
I just update xstate from 5.7.0 to 5.8.0 and get this error:
`
Type 'ActionFunction<{ oldQuery: Record<string, any>; query: Record<string, any>; }, EventObject, AnyEventObject, NonReducibleUnknown, never, never, never, string>' is not assignable to type 'ActionFunction<{ oldQuery: Record<string, any>; query: Record<string, any>; }, AnyEventObject, AnyEventObject, NonReducibleUnknown, never, Values<...>, never, never>'.
  Types of property '_out_TDelay' are incompatible.
    Type 'string | undefined' is not assignable to type 'undefined'.
      Type 'string' is not assignable to type 'undefined'.ts(2322)
`

And here is the code:
<img width="589" alt="image" src="https://github.com/statelyai/xstate/assets/126638137/f33bc3ea-8412-4bf9-9e9a-b9160ca53789">

If I change `sendParent` to `sendTo(SpecialTarget.Parent)`, it works.
It is strange, because `sendParent` use `sendTo(SpecialTarget.Parent)` internal.

Please help me to resolve this

**Comment by Andarist** [comment]
The fix on our side would be to add `TUsedDelay` to `sendParent`

**Reply by Andarist** [reply, marked as answer]
I've opened a PR with the fix here: https://github.com/statelyai/xstate/pull/4772
